### PR TITLE
Add UnoCSS Detection

### PR DESCRIPTION
**Pull request by ysm-dev**
website:
- https://uno.antfu.me/

examples:
- https://weak-rabbit-42.deno.dev/
- https://dash.deno.com/
- https://vitesse.netlify.app/
- https://pdf.ruleplaying.com/
- https://springue.github.io/demo-playground/

### CODE REVIEW

File: src/technologies/u.json [hunk 1]
```diff
@@ -773,6 +773,15 @@
     "saas": true,
     "website": "https://ulogin.ru"
   },
+  "UnoCSS": {
+    "cats": [
+      66
+    ],
+    "css": "--un-(?:rotate|translate|space-x|text-opacity|border-opacity)",
+    "description": "UnoCSS is instant on-demand Atomic CSS engine.",
+    "icon": "UnoCSS.svg",
```
Comment (nurbek91) [inline src/technologies/u.json:782]: 32x32
[UnoCSS.svg.zip](https://github.com/wappalyzer/wappalyzer/files/9551128/UnoCSS.svg.zip)

Comment (nurbek91) [reply src/technologies/u.json:782]: "oss": true,

File: src/technologies/u.json [hunk 2]
```diff
@@ -773,6 +773,15 @@
     "saas": true,
     "website": "https://ulogin.ru"
   },
+  "UnoCSS": {
+    "cats": [
+      66
+    ],
+    "css": "--un-(?:rotate|translate|space-x|text-opacity|border-opacity)",
```
Comment (nurbek91) [inline src/technologies/u.json:780]: image

```
 "dom": {
      "style[data-unocss]": {
        "attributes": {
          "data-unocss": "([\\d\\.]+)\\;version:\\1"
        }
      }
    },
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!
Need small updates maybe

File: src/technologies/u.json (hunk 1 above)
Comment (ysm-dev) [reply src/technologies/u.json:790]: [b60f702](https://github.com/wappalyzer/wappalyzer/pull/6753/commits/b60f702b3701efb278d90828419c35e16a657b4c)

[f35e147](https://github.com/wappalyzer/wappalyzer/pull/6753/commits/f35e147a247c730b9e8e4cdc054da303f81604e2)

File: src/technologies/u.json (hunk 2 above)
Comment (ysm-dev) [reply src/technologies/u.json]: [79fcc72](https://github.com/wappalyzer/wappalyzer/pull/6753/commits/79fcc727045d861ccdcde5560fab09ad0851e433)

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**